index.html

```diff
@@ -49,10 +49,193 @@ <h2>LET'S CONNECT</h2>
+                    </li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+        <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio(1).png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">TONIC </h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li>
+                      <div class="spans"></div>
+                    </li>
+                    <li>Backend Dev</li>
+                    <li>
+                      <p class="spans"></p>
+                    </li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+        <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio2.png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">Multi Post stories </h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li>
+                      <div class="spans"></div>
+                    </li>
+                    <li>Backend Dev</li>
+                    <li>
+                      <p class="spans"></p>
+                    </li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+        </section>
+
+
+        
+         <section class="container-b">
+          <div class="bg-b">
+                <h1 class="about-h1">About Myself</h1>
```

Comment:
- [Optional] Kindly note that it is best SEO practice to have just one h1 tag in your document. This is so screenreaders can identify the main title/heading of your document. Kindly change this to any other heading element `h2 --- h5`.